crates/forge/bin/cmd/install.rs

```diff
@@ -236,6 +236,9 @@ impl Installer<'_> {
         // checkout the tag if necessary
         self.git_checkout(&dep, path, false)?;
 
+        trace!("updating dependency submodules recursively");
+        self.git.root(path).submodule_update(false, false, false, true, None::<PathBuf>)?;
```

Comment:
I'd prefer this if it compiles
```suggestion
        self.git.root(path).submodule_update(false, false, false, true, std::iter::empty::<PathBuf>())?;
```